package com.nuvio.app.core.ui

import coil3.ComponentRegistry
import coil3.ImageLoader

internal actual fun ImageLoader.Builder.configurePlatformImageLoader(): ImageLoader.Builder = this

internal actual fun ComponentRegistry.Builder.addPlatformImageComponents(): ComponentRegistry.Builder = this
